en/arrays.texy

```diff
@@ -129,4 +129,101 @@ Arrays::renameKey($array, 'first', 'renamed');
 // $array = ['renamed' => 10, 'second' => 20];
 \--
 
-{{composer: nette/utils}}
+
+flatten(array $array, bool $preserveKeys = false): array .[method]
+------------------------------------------------------------------
+
+Transform multidimensional array to flat array.
+
+/--php
+$array = Arrays::flatten([
+	2 => ['a', ['b']],
+	4 => ['c', 'd'],
+	'e',
+]);
+// $array = [1 => 'a', 2 => 'b', 3 => 'c', 4 => 'd'];
+\--
+
+
+isList(array $array): bool .[method]
+------------------------------------
+
+Check if the array is indexed by numeric keys (as list).
+
+/--php
+Arrays::isList(['a', 'b', 'c'])); // true
+Arrays::isList([4 => 1, 2, 3])); // false
+\--
+
+
+normalize(array $arr, string $filling = null): array .[method]
+--------------------------------------------------------------
+
+Replace numeric keys in array to the value from variable `$filling`.
+
+/--php
+$array = Arrays::normalize([1 => 'first', 'a' => 'second']);
```

Comment:
I think that I don't understand what you mean :/ I was started from czech version: https://github.com/nette/docs/blob/doc-2.4/cs/arrays.texy#L160 ... what do you suggest?